scripts/docker-build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
```

Comment:
If we are to remove the `build-image-plugins` repository, we should transfer the few issues to this repository: https://github.com/netlify/build-image-plugins/issues